src/component/Todo.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import TodoForm from '../component/TodoForm';
+import TodoItem from '../component/TodoItem';
+import Todos from '../mock/data';
+
+class Todo extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      id: '',
+      content: '',
+    }
+  }
+
+  // it sets the value on any change
+   onChange = (event) => {
+    event.preventDefault();
+    this.setState({ [event.target.name]: event.target.value, id: Todos.length + 1 })
```

Comment:
Why are you increasing the number of todos everytime the user types?

Reply:
Because in my mock data, I have ID. so I want to assign Id to the new created task depending on how many data I have in my array.